### ISSUE TITLE
Radio buttons in Advanced Search with same name attributes must be part of a group

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1071]
Violation: Radio inputs with the same name attribute value `name="op2"` and `name="op3"` must be part of a group.

Ensure all related radio buttons are grouped together using:
- `<fieldset>` and `<legend>` tags
- ARIA groups role="group" or role="radiogroup" and aria-label or aria-labelledby
- aria-labelledby that points to the same element for every radio button in the group
